packages/arb-token-bridge-ui/src/hooks/TransferPanel/useGasSummary.ts

```diff
@@ -72,15 +66,15 @@ export function getGasSummaryStatus({
     return 'unavailable'
   }
 
-  if (balance === null || oftFeeSummaryLoading) {
+  if (balance === null) {
```

Comment:
Again, not sure about this whole change since fetching OFT-fee-estimates will be required to block the transfer until we properly fetch it.
What needs to be displayed in summary has to be -
`OFT fee estimate (derived from quoteOFT) + Signing gas fee estimate (derived from signer.getGasEstimate)`

Reply:
`useGasEstimate` now use `OftV2TransferStarter` to estimate gas, so it's a blocker, but now it's transparent, we don't need a new flag